common/scala/src/main/scala/whisk/common/TransactionId.scala

```diff
@@ -241,12 +245,15 @@ object TransactionId {
  * A thread-safe transaction counter.
  */
 trait TransactionCounter {
-  val numberOfInstances: Int
+  case class TransactionCounterConfig(strides: Int)
+
+  val transCounterConfig = loadConfigOrThrow[TransactionCounterConfig](ConfigKeys.transactions)
+  val strides = transCounterConfig.strides
   val instanceOrdinal: Int
 
-  private lazy val cnt = new AtomicInteger(numberOfInstances + instanceOrdinal)
+  private lazy val cnt = new AtomicInteger(strides + instanceOrdinal)
```

Comment:
Seems that having the instance ordinal and stride is duplicitous. Can we reduce to just strides